Win/Mineways.cpp

```diff
@@ -1158,6 +1177,8 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
 
                 return 0;
             }
+
+			sprintf_s(gSkfbPData.skfbName, "%ws", stripWorldName(gWorld));
```

Comment:
It is necessary to initialize sketchfab model name field with the world name by default